core/Dockerfile

```diff
@@ -3,7 +3,7 @@ FROM python:3.6-alpine
 MAINTAINER Abhilash Raj
 
 #Add startup script to container
-ADD assets/run.sh /opt/run.sh
+COPY docker-entrypoint.sh /usr/local/bin/
```

Comment:
From https://docs.docker.com/engine/userguide/eng-image/dockerfile_best-practices/#add-or-copy:
> Although ADD and COPY are functionally similar, generally speaking, COPY is preferred.

It's separated as commit, so I can postpone that if you disagree. I have used the official `docker-library` convention for naming entrypoints. Renaming it also saves a directory :P